NGS2_WITNESS_Cycle1_confirmatory_exp1.R

```diff
@@ -0,0 +1,73 @@
+#Created by Pablo Diego Rosell, PhD, for Gallup inc. in March 2017
+
+# Load data for analysis
+exp1_cooperation <- read.csv('NGS2-Cycle1-Experiment1/cooperation_exp1.csv',
+                             header = TRUE, sep = ',')
+exp1_rewire <- read.csv('NGS2-Cycle1-Experiment1/rewire_exp1.csv', header = TRUE,
+                        sep = ',')
+
+### Show stopper
+
+print("ATENTION!! Need to make sure the exp1_cooperation dataset includes the following variables: sessionnum, condition, decision..0.D.1.C.")
+
+#Hypothesis 4.1.1: Fixed network compositions reduce exp1_cooperation
+#Rand  et al. (2011) coeff = -0.186, p<.000
+
+exp1_cooperation.fixed<-subset(exp1_cooperation, condition=="Static")
+logit.fixed <- glm(decision..0.D.1.C.~ round_num, data = exp1_cooperation.fixed, family = "binomial")
+logit.multiwayvcov.fixed <- cluster.vcov(logit.fixed, cbind(exp1_cooperation.fixed$sessionnum, exp1_cooperation.fixed$playerid))
+coeftest(logit.fixed, logit.multiwayvcov.fixed)

```

Comment:
There is a lot of repeated code here. This should be function-ized so code can be reused. The following could be used for the first three blocks; with some additional tweaks, it can easily incorporate the fourth block as well:
```
logit_test_and_summary <- function(data, condition) {
    sub <- subset(data, condition == condition)
    logit_result <- glm(decision..0.D.1.C. ~ round_num, data = sub, family = 'binomial')
    logit_vcov <- cluster.vcov(logit_result, cbind(sub$sessionnum, sub$playerid))
    print(coeftest(logit_result, logit_vcov))
}
```

This could then be called with:
```
logit_test_and_summary(exp1_cooperation, 'Static')
logit_test_and_summary(exp1_cooperation, 'Random')
```

and so on. However, this might be off by a bit; it's seems like none of the intermediate variables are re-used later on nor or they inspected, so it seems like what's important is to see the `coeftest()` result, which should be sufficient to wrap inside `print()` in the above function. I think this will help make the code a bit more readable.

Reply:
So true! I tweaked your proposed function a little bit to avoid "condition == condition" which throws R off, but working great otherwise, thanks!!